config/routes.rb

```diff
@@ -1,10 +1,22 @@
 # frozen_string_literal: true
 # rubocop:disable Metrics/BlockLength
 
+def rake_task?(tasks = [])
+  defined?(Rake) && tasks.any? do |task|
+    Rake.application.top_level_tasks.include?(task)
```

Comment:
Oh and please wrap this line in a `# :nocov:` environment since it's impossible to (reasonably) execute it in a test.